packages/flutter/test/widgets/draggable_scrollable_sheet_test.dart

```diff
@@ -7,44 +7,55 @@ import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
+  // Allowable error to still consider floats equal.
+  const double epsilon = 1e-6;
```

Comment:
Please avoid introducing new epsilons other than the one in `constants.dart` from foundation.